model/database/deprecated/registry-deprecated.yaml

```diff
@@ -489,8 +489,8 @@ groups:
               stability: development
             - id: postgresql
               value: "postgresql"
-              brief: "PostgreSQL (This value has stability level RELEASE CANDIDATE)"
-              stability: release_candidate
+              brief: "PostgreSQL"
+              stability: development
```

Comment:
Do you think we should downgrade to development when we deprecate things?
We wouldn't do this for stable ones (to keep code-gen producing the same code), so I wonder if we need to do it for other stability levels.

Reply:
I don't mind us doing it now for this change, created https://github.com/open-telemetry/semantic-conventions/issues/2156 to track the general question